backend/src/users/users.controller.ts

```diff
@@ -40,8 +40,24 @@ export class UsersController {
     }
 
     @Get()
```

Comment:
You should use `@Get(':token')` for this request to differentiate between `findByToken` and `findAllUsers`.